packages/core/admin/admin/src/components/Form.tsx

```diff
@@ -670,45 +670,16 @@ const useField = <TValue = any,>(path: string): FieldValue<TValue | undefined> =
 
   const handleChange = useForm('useField', (state) => state.onChange);
 
-  const formatNestedErrorMessages = (stateErrors: FormErrors<FormValues>) => {
```

Comment:
I don't think we can remove this logic. The problem is for cases like this:


```
 state.errors =
	{
	    "Compo.Nested.Unique": "This attribute must be unique"
	}
```

When you just do `const error = getIn(state.errors, path);` we can't extract and highlight these nested errors correctly